README.ru.md

```diff
@@ -0,0 +1,68 @@
+# sway
+
+sway - это [i3](https://i3wm.org/)-совместимый композитор [Wayland](http://wayland.freedesktop.org/).
```

Comment:
Not sure if "композитор" is a good choice, how about "композитный менеджер"? This term is used in [Wayland  wikipedia page](https://ru.wikipedia.org/wiki/Wayland).

Reply:
Hmm, I'm not sure "композитный" is a good word here. It seems to mean "that is composited (of something)" rather than "it composits" [1] [2].

[1]: https://ru.wiktionary.org/wiki/%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%B7%D0%B8%D1%82%D0%BD%D1%8B%D0%B9
[2]: https://dic.academic.ru/dic.nsf/es/159787/%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%B7%D0%B8%D1%82%D0%BD%D1%8B%D0%B9